src/test/java/com/fastcampus/projectboard/domain/AuditingFieldsEqualsHashcodeTest.java

```diff
@@ -0,0 +1,81 @@
+package com.fastcampus.projectboard.domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.random.RandomGenerator;
+import java.util.random.RandomGeneratorFactory;
+
+import static com.fastcampus.projectboard.domain.AuditingFieldsReflectionHelper.setAuditingFields;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("[AuditingField] AuditingFields equals hashcode 메서드 테스트")
+public class AuditingFieldsEqualsHashcodeTest {
+
+    private LocalDateTime now;
+
+    @BeforeEach
+    void timeInit() {
+        now = LocalDateTime.now();
+    }
+
+    @DisplayName("서로다른 5개의 객체 중복 비교")
+    @Test
+    void givenArticles_whenDistinct_thenNotDuplication() {
+        // given & when
+        List<Article> articles = createDiffFiveArticles();
+        Set<Article> removeDuplicationArticle = new HashSet<>(articles);
+
+        // then
+        assertThat(articles.size()).isEqualTo(removeDuplicationArticle.size());
+    }
+
+    @DisplayName("리스트에 모든 객체가 중복일 때 객체 중복 비교")
+    @Test
+    void givenSameOffsetArticles_whenDistinct_thenRemoveDuplicate() {
+        // Given & When
+        int offset = 5;
+        List<Article> articles = List.of(createArticle(offset), createArticle(offset), createArticle(offset));
+        Set<Article> removeDuplicationArticles = new HashSet<>(articles);
+
+        // Then
+        assertThat(articles.size()).isNotEqualTo(removeDuplicationArticles.size());
+        assertThat(1).isEqualTo(removeDuplicationArticles.size());
+    }
+
+    @DisplayName("모든 필드값이 같은 2개의 객체 중복 비교")
+    @RepeatedTest(10)
+    void givenTwoSameArticle_when_thenSameArticle() {
+        // given
+        RandomGenerator randomGenerator = RandomGeneratorFactory.getDefault().create();
+        int offset = randomGenerator.nextInt(100);
```

Comment:
리뷰에 설명드렸지만 이 코드가 아마 다음과 차이가 없을 겁니다. 여기는 팩토리를 사용할 동기가 좀 부족합니다.

```suggestion
        int offset = RandomGenerator.getDefault().nextInt(100);
```

Reply:
해당 사항은 바로 Commit suggestion으로 반영했습니다 감사합니다.